ckanext/datastore/backend/postgres.py

```diff
@@ -1631,12 +1657,17 @@ def delete_data(context: Context, data_dict: dict[str, Any]):
                                              fields_types, query_dict)
 
     where_clause, where_values = _where(query_dict['where'])
-    sql_string = u'DELETE FROM "{0}" {1}'.format(
+    sql_string = u'DELETE FROM "{0}" {1} {2}'.format(
         data_dict['resource_id'],
```

Comment:
While we're here let's make this safer
```suggestion
    sql_string = u'DELETE FROM {0} {1} {2}'.format(
        identifier(data_dict['resource_id']),
```